### ISSUE TITLE
The 0.3 release doesn't have any notable changes

### ISSUE THREAD

**Issue by kylone** [jswrenn/typic#12]
I was looking at the changelog in CHANGELOG.md, and saw that there was no release notes for the most recent version: 0.3. What changed required a new minor version in this release?

**Comment by jswrenn** [comment]
Good catch! The design of stability declaration system was totally redesigned: https://github.com/jswrenn/typic/pull/5

It's still a work-in-progress, unfortunately. There's just enough there to validate the design, but my attention has since shifted to preparing an RFC for safe transmutation: https://github.com/rust-lang/project-safe-transmute/pull/5
